extensions/node-js-evaluate/src/index.tsx

```diff
@@ -0,0 +1,127 @@
+import {List, ActionPanel, Detail, CopyToClipboardAction, PushAction, preferences} from "@raycast/api";
+import {useEffect, useState} from "react";
+
+interface State {
+    query?: string;
+    result?: string;
+    type?: string;
+    error?: Error;
+    isLoading: boolean
+}
+
+const fastEvaluation = preferences?.fastEvaluation?.value ?? false;
+
+export default function Command() {
+    const [state, setState] = useState<State>({isLoading: false});
```

Comment:
Hi.

I saw it once.
The reason the action bar is blinking is because... 

The root component data of the SlowEval and FastEval functions has changed, so it is a phenomenon of drawing a new screen.

Right here.

Reply:
Thanks @kkange!
I'm going to rewrite that in a whole different and nicer manner :)